scripts/valhalla_build_timezones

```diff
@@ -23,225 +23,13 @@ rm -f ./timezones-with-oceans.shapefile.zip
 
 url="https://github.com/evansiroky/timezone-boundary-builder/releases/download/2018d/timezones-with-oceans.shapefile.zip"
```

Comment:
@nilsnolde should we use the latest release?  https://github.com/evansiroky/timezone-boundary-builder/releases

Reply:
Jep @gknisely, gut that'll take 1-2 more PRs. We need to also update the IANA files.

Next PR I want to make one more refactor so we can keep the current tz order to not break existing clients after we update tzdb.